Traceback (most recent call last):
  File "/home/wang/distribution/bounded_malware_classification/drebin/src/Main.py", line 62, in <module>
    main(ParseArgs(), True)
    ~~~~^^^^^^^^^^^^^^^^^^^
  File "/home/wang/distribution/bounded_malware_classification/drebin/src/Main.py", line 38, in main
    RandomClassification(dual, penalty, train_years, True, os.path.join(dir, "training", "malware"), os.path.join(dir, "training", "goodware"), TestSize, FeatureOption, Model, NumFeatForExp, os.path.join(dir, "training"))
    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wang/distribution/bounded_malware_classification/drebin/src/RandomClassification.py", line 113, in RandomClassification
    all_parameters = np.prod(BestModel.coef_.shape) + BestModel.intercept_.size
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'float' object has no attribute 'size'
usage: Main.py [-h] [--datadir DATADIR] [--ncpucores NCPUCORES]
               [--testsize TESTSIZE] [--model MODEL]
               [--numfeatforexp NUMFEATFOREXP] [--penalty PENALTY]
               [--dual DUAL] [--priorPortion PRIORPORTION] [--future FUTURE]
               [--eta ETA] [--mu MU] [--apk APK] [--i I]
Main.py: error: argument --future: expected one argument
